### ISSUE TITLE
Obtaining body names for ASCII STLs

### ISSUE THREAD

**Issue by aerogt3** [mikedh/trimesh#1555]
I've noticed it's possible to get the number of solids/bodies in ASCII STLs 
`mesh.body_count `

Is it possible to get the **names** of the bodies as well? This can be achieved with grep, but would be cleaner to do via trimesh directly.

**Comment by ReubenVandezande** [comment]
Yes, assuming you have an ASCII multi-body STL, `trimesh` will return a `Scene` object. The bodies are stored in key/value pairs in the `geometry` attribute where the key is the body name. 

```
mesh.geometry
OrderedDict([('body1', <trimesh.Trimesh(vertices.shape=(1000, 3), faces.shape=(3000, 3))>), ('body2', <trimesh.Trimesh(vertices.shape=(2000, 3), faces.shape=(5000, 3))>)])
```
